examples/glue/glue_entrypoint.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 🚪 GLUE Entrypoint\n",
+    "\n",
+    "This notebook is intended to demonstrate how to use the [GLUE (General Language Understanding Evaluation)](https://gluebenchmark.com/)  entrypoint for pretraining NLP models and finetuning them on the 8 GLUE tasks. This will cover:\n",
```

Comment:
```suggestion
    "This notebook is intended to demonstrate how to use the [GLUE (General Language Understanding Evaluation)](https://gluebenchmark.com/) entry point for pre-training and fine-tuning NLP models across the 8 GLUE tasks.\n This will cover:\n",
```